gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 # keep sorted please, use Edit -> Sort Lines
 
 [versions]
-androidx-lifecycle = "2.4.1"
+androidx-lifecycle = "2.5.1"
```

Comment:
I will fix this file depending on what the outcome of this proposal is.
We should update these dependencies in another PR. Without updating them, hilt causes issues.